### PR TITLE
Update to Rocky Linux yaml for Version 9.0

### PR DESCRIPTION
**Pull request by sspencerwire**
This is to enable version 9.0 of Rocky Linux.  Hopefully, it all looks good, if not, please flag me, email me, etc.

### CODE REVIEW

File: images/rockylinux.yaml [hunk 1]
```diff
@@ -35,6 +35,41 @@ source:
     JmeWf4VZyebm+gc50szsg6Ut2yT8hw==
     =AiP8
     -----END PGP PUBLIC KEY BLOCK-----
+    
+    # RPM-GPG-KEY-rockylinux-9
+    - |-
```
Comment (monstermunchkin) [inline images/rockylinux.yaml:40]: I believe this indentation is wrong, and needs to be aligned correctly (needs to be moved to the left).

**Review by monstermunchkin — CHANGES_REQUESTED** [review]

File: images/rockylinux.yaml (hunk 1 above)
Comment (sspencerwire) [reply images/rockylinux.yaml]: I've moved it see https://github.com/lxc/lxc-ci/pull/576/commits/a6ac9a6482b3dd08813db7c72a5c8a1928e0a9af

**Comment by sspencerwire** [comment]
@monstermunchkin Thanks for your reviews and suggested fixes. I'm not a dev but passionate about LXD and Rocky Linux. My problem, in this case, is that I don't know how to accomplish this: 

  "Also, please don't use a separate commit for fixing the alignment. Just merge it with the commit which introduced the change." 

I could close the PR and re-issue with all of the fixes if that is acceptable. Let me know.

**Comment by stgraber** [comment]
Did it for you

**Comment by monstermunchkin** [comment]
@stgraber the gpg key is still not indented correctly. We used 4 spaces for the previous key.

**Review by monstermunchkin — APPROVED** [review]

**Comment by sspencerwire** [comment]
Thankyou @monstermunchkin and @stgraber ! I'll do better next time.

**Pull request merged**